nordicsemi/__main__.py

```diff
@@ -505,6 +559,14 @@ def pkg():
               help='The private (signing) key in PEM fomat.',
               required=False,
               type=click.Path(exists=True, resolve_path=True, file_okay=True, dir_okay=False))
+@click.option('--boot-validation',
```

Comment:
As it is now boot validation in packages must be done for either both SD and APP, or none of them. Is that ok, or should it be possible to use boot validation for only one of them?

Reply:
I guess that is only relevant for SD+APP updates? Preferably, you should be able to specify for each, but it's not critical IMO.

Reply:
If there is no specific use case for setting them separately, it might just make it easier for the user keeping them together. But right now package generation and settings generation do this differently, as settings generation handles SD and APP boot validation separately. I guess they should behave in the same way?

Reply:
Updated package to have one argument for SD and one argument for APP boot validation.